src/torchmetrics/functional/__init__.py

```diff
@@ -63,6 +63,7 @@
 from torchmetrics.functional.regression.log_mse import mean_squared_log_error
 from torchmetrics.functional.regression.mae import mean_absolute_error
 from torchmetrics.functional.regression.mape import mean_absolute_percentage_error
+from torchmetrics.functional.regression.minkowski import minkowski_distance
```

Comment:
please also add such an entry to the `functional/regression/__init__.py` file :]